cogs/games.py

```diff
@@ -20,18 +20,203 @@
 # discord-py requirements
 import discord
 from discord.ext import commands
+from discord import Embed
```

Comment:
I wouldn't do this, you only use Embed once in your code so its fine to use discord.Embed for it, and this is how it is done in every other cog. embed is also a commonly used variable name so this would be a bit confusing when people write other commands in this cog